pkg/pf/tests/internal/cross-tests/configure.go

```diff
@@ -22,7 +22,7 @@ import (
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-framework/provider"
-	"github.com/hashicorp/terraform-plugin-framework/provider/schema"
+	pschema "github.com/hashicorp/terraform-plugin-framework/provider/schema"
```

Comment:
Let's not do this. `pschema` often means Pulumi Schema, and we should package name overrides when possible unless they are standardized.

Reply:
pschema is used for provider schema throughout the PF code - I've renamed this to match the rest of the codebase.

The clash is unfortunate but I think we should address that seprately - in the PF code pschema means provider schema